packages/database/src/core/RepoManager.ts

```diff
@@ -80,7 +83,10 @@ export class RepoManager {
    * @return {!Database}
    */
   databaseFromApp(app: FirebaseApp, url?: string): Database {
-    const dbUrl: string = url || app.options[DATABASE_URL_OPTION];
+    const dbUrl: string =
+      url ||
+      app.options[DATABASE_URL_OPTION] ||
+      'http://' + process.env[FIREBASE_DATABASE_EMULATOR_HOST_VAR];
```

Comment:
If the environment variable is not set, we're going to end up with `dbUrl = "http://undefined"`, which seems pretty surprising.

Reply:
Thanks for catching that.